docs/fsharp/style-guide/formatting.md

```diff
@@ -116,657 +62,512 @@ This is sometimes called “vanity alignment” or “vanity indentation”. The
 
 ```fsharp
-module MyAPI =
-    let publicFunctionOne param1 param2 param2 = ...
+// OK
+someTupledFunction (
+    478815516,
+    "A very long string making all of this multi-line",
+    1515,
+    false
+)
 
-    let publicFunctionTwo param1 param2 param3 = ...
+// OK, but formatting tools will reformat to the above
+someTupledFunction
+    (478815516,
+     "A very long string making all of this multi-line",
+     1515,
+     false)
 ```
 
-### Use camelCase for internal and private module-bound values and functions
+If argument expressions are short, separate arguments with spaces and keep it in one line.
 
-Use camelCase for private module-bound values, including the following:
-
-* Ad hoc functions in scripts
+```fsharp
+let person = new Person(a1, a2)
 
-* Values making up the internal implementation of a module or type
+let myRegexMatch = Regex.Match(input, regex)
 
-```fsharp
-let emailMyBossTheLatestResults =
-    ...
+let untypedRes = checker.ParseFile(file, source, opts)
 ```
 
-### Use camelCase for parameters
-
-All parameters should use camelCase in accordance with .NET naming conventions.
+If argument expressions are long, use newlines and indent one level, rather than indenting to the left-parenthesis.
 
 ```fsharp
-module MyModule =
-    let myFunction paramOne paramTwo = ...
-
-type MyClass() =
-    member this.MyMethod(paramOne, paramTwo) = ...
-```
-
-### Use PascalCase for modules
+// OK
+let person =
+    new Person(
+        argument1,
+        argument2
+    )
 
-All modules (top-level, internal, private, nested) should use PascalCase.
+// OK
+let myRegexMatch =
+    Regex.Match(
+        "my longer input string with some interesting content in it",
+        "myRegexPattern"
+    )
 
-```fsharp
-module MyTopLevelModule
+// OK
+let untypedRes =
+    checker.ParseFile(
+        fileName,
+        sourceText,
+        parsingOptionsWithDefines
+    )
 
-module Helpers =
-    module private SuperHelpers =
-        ...
+// Not OK
+let person =
+    new Person(argument1,
+               argument2)
 
-    ...
+// Not OK
+let untypedRes =
+    checker.ParseFile(fileName,
+                      sourceText,
+                      parsingOptionsWithDefines)
 ```
 
-### Use PascalCase for type declarations, members, and labels
-
-Classes, interfaces, structs, enumerations, delegates, records, and discriminated unions should all be named with PascalCase. Members within types and labels for records and discriminated unions should also use PascalCase.
+The same rules apply even if there is only a single multi-line argument.
 
 ```fsharp
-type IMyInterface =
-    abstract Something: int
-
-type MyClass() =
-    member this.MyMethod(x, y) = x + y
-
-type MyRecord = { IntVal: int; StringVal: string }
+let poemBuilder = StringBuilder()
+poemBuilder.AppendLine(
+    """
+The last train is nearly due
+The Underground is closing soon
+And in the dark, deserted station
+Restless in anticipation
+A man waits in the shadows
+    """
```

Comment:
Was this the intended indentation?

```suggestion
    """
    The last train is nearly due
    The Underground is closing soon
    And in the dark, deserted station
    Restless in anticipation
    A man waits in the shadows
    """
```

Reply:
No, this is a triple-quote string, it's intentional to left-indent